source/frontend/src/features/leases/detail/LeasePages/insurance/edit/InsuranceForm.tsx

```diff
@@ -27,47 +28,24 @@ const InsuranceForm: React.FunctionComponent<React.PropsWithChildren<IInsuranceR
     <FormSection className="mb-3" data-testid="insurance-form">
       <SubTitle data-testid="insurance-form-title">{insuranceType.description}</SubTitle>
       {insuranceType.id === 'OTHER' && (
-        <Row>
-          <Col md="4">
-            <div>
-              <strong>Insurance Type:</strong>
-            </div>
-            <Input field={withNameSpace(nameSpace, 'otherInsuranceType')} />
-          </Col>
-        </Row>
+        <SectionField label="Insurance type" labelWidth="3">
+          <Input field={withNameSpace(nameSpace, 'otherInsuranceType')} />
+        </SectionField>
```

Comment:
Thanks for doing this but I've got a PR in review to fix clean up this UI as part of **PSP-7493**
To avoid merge conflicts on your side let's try to merge your changes first so I can deal with conflicts on my end

Reply:
Thank you for letting me know. I was doubting if I should clean up as I go or should I leave it as it was. I'll consider this incident for next PRs.